package.json

```diff
@@ -17,50 +17,25 @@
   "types": "./dist/main.d.ts",
   "scripts": {
     "dev": "vite",
-    "preview": "vite preview",
     "build": "npm run lint && vite build",
     "build:lib": "vite build",
     "lint": "eslint . --max-warnings=0 && vue-tsc --noEmit",
     "prepublishOnly": "npm run build",
     "prepare": "npm run build",
     "test:unit": "vitest run --dom",
-    "cy:e2e:headless": "cypress run --e2e",
-    "cy:e2e": "cypress run --e2e --headed",
-    "test:e2e": "start-server-and-test dev http://localhost:8080 cy:e2e",
-    "test:e2e:headless": "start-server-and-test dev http://localhost:8080 cy:e2e:headless",
-    "web:open": "vite --open",
-    "web:release": "npm run lint && vite build",
-    "native:build": "node ./scripts/vite_build_for_native.cjs",
-    "android:copy": "npm run native:build && npx cap copy android",
-    "android:copy:release": "npm run native:build && npx cap copy android",
-    "android:open": "npm run android:copy && npx cap open android",
-    "ios:copy": "npm run native:build && npx cap copy ios",
-    "ios:copy:release": "npm run native:build && npx cap copy ios",
-    "ios:open": "npm run ios:copy && npx cap open ios",
-    "electron:copy": "npm run native:build && npx cap copy @capacitor-community/electron",
-    "electron:open": "npm run electron:copy && npx cap open @capacitor-community/electron",
-    "electron:release": "npm run electron:copy && cd electron && npm run electron:release",
-    "electron:debug": "npm run electron:copy && cd electron && npm run electron:start",
     "prettier:check": "prettier . --check",
     "prettier:fix": "prettier . --write"
   },
   "dependencies": {
-    "@capacitor/app": "5.0.7",
-    "@capacitor/core": "5.7.4",
-    "@capacitor/haptics": "5.0.7",
-    "@capacitor/keyboard": "5.0.8",
-    "@capacitor/status-bar": "5.0.7",
     "@ionic/vue": "^7.0.0",
-    "@ionic/vue-router": "^7.0.0",
-    "@types/luxon": "^3.3.4",
```

Comment:
is it not useful since we use luxon?

Reply:
It is but I moved it to the `devDependencies`. I don't see why it would be needed apart from dev.